tests/test_amqp.py

```diff
@@ -1261,3 +1262,17 @@ def test_message_info(self):
         )
 
         self.assertDictEqual(info, msg.info())
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize('url,kwargs,exp', [
+    ('amqps://', {}, {'ssl': True}),
+    ('localhost', {'ssl': True}, {'ssl': True}),
```

Comment:
if #136 would be merged the 'localhost' could be removed as it is default host name